### ISSUE TITLE
Fuji X-M1 - can't get less noise in shadows...

### ISSUE THREAD

**Issue by serhiykobyakov** [jcelaya/hdrmerge#78]
Hi,

Trying to merge few RAW's in hdrmerge. Canon raws are combined well, no noise in shadows in resulting dng.

But I can't produce the same result with Fujifilm X-M1

![test](https://cloud.githubusercontent.com/assets/15909653/11249156/969eb62e-8e25-11e5-9127-8d090ea74b71.jpg)

**Comment by serhiykobyakov** [comment]
I made two sample RAWs with 2eV difference for testing purposes: https://www.dropbox.com/s/t9bzo1j8hiutrzr/x-m1_bracket.zip?dl=0

**Comment by serhiykobyakov** [comment]
Furthermore, in GUI there is a problem with brightness. when I slide it to brighter - preview becomes completely black, and after that I cannot bring back the image, the preview just remains black...

**Comment by sguyader** [comment]
This issue seems to be related to the way black levels are read by libraw for X-trans sensors, I think.
I have the same issue with the X-T2: preview is black, and the output DNG is pinkish. More correct colors can be retrieved by tweaking black levels in RawTherapee, but the resulting image is noisy.
In fact, in file `RawParameters.cc` I replaced `black=r.color.cblack;` with `black=1022;` and this gives me an output dng with correct black levels and colors, but what I see is that in fact, only the darkest frame of the bracket is used and its brightness amplified, this is why the output is noisy.

**Comment by Beep6581** [comment]
@serhiykobyakov the download link is 404.

**Comment by sguyader** [comment]
@Beep6581 Morgan, the issue is almost 2 years old...
In case you wanna check, I have uploaded some files [there](https://filebin.net/lag66zdqpx22jzzi) (3 RAF files, and DNG output from HDRMerge).

**Comment by Beep6581** [comment]
Confirmed. Compiled against libraw-0.18.5.

For reference, the white level from `camconst.json` for the FUJIFILM X-T2 is 16100, and the black level in the metadata is 1022.

**Comment by Beep6581** [comment]
Closing as duplicate of/depending on #112
